web/src/message_list_view.js

```diff
@@ -1013,11 +1051,8 @@ export class MessageListView {
             }
         };
 
-        if (message_containers.length === 0) {
-            return undefined;
-        }
```

Comment:
Do I understand correctly that this case should have been impossible, since we check if `messages.length` is 0 earlier, and it should have the same length?

Reply:
Oh yeah sorry I meant to add some comments for reviewers that I had on my private WIP PR, this is one of them. You're right. We're changing to using `message`s instead of `message_container`s, and we already checked `messages length` earlier so we don't need to check it again here.